api/api.py

```diff
@@ -35,7 +35,7 @@
     'uid': '[0-9a-zA-Z.@_-]*',
 
     # Container name
-    'cname': 'projects|sessions|acquisitions|collections',
+    'cname': 'projects|sessions|acquisitions|collections|analyses',
```

Comment:
It just occurred to me now, that removal would make this a breaking change, forcing UI to switch immediately. Deferring judgement.